src/cbc_sdk/platform/vulnerability_assessment.py

```diff
@@ -51,6 +51,24 @@ def __init__(self, cb, model_unique_id, os_product_id=None, initial_data=None):
             self._refresh()
         self._full_init = True
 
+    def _subobject(self, name):
+        """
+        Returns the "subobject value" of the given attribute.
+
+        Args:
+            name (str): Name of the subobject value to be returned.
+
+        Returns:
+            Any: Subobject value for the attribute, or None if there is none.
+        """
+        if name == "os_info" and self._info["os_info"]:
```

Comment:
Is it possible for the key not to be present at all.

Reply:
Whoops I didn't mean to include this piece. I was brainstorming something separate